modules/consensus/tendermint/prover/src/tests/integration_tests.rs

```diff
@@ -0,0 +1,261 @@
+#[cfg(test)]
+mod tests {
+	use std::time::{SystemTime, UNIX_EPOCH};
+
+	use crate::{prove_header_update, Client, CometBFTClient, HeimdallClient};
+	use tendermint_verifier::{TrustedState, VerificationError, VerificationOptions};
+	use tracing::trace;
+
+	const STANDARD_RPC_URL: &str =
+		"https://rpc.ankr.com/sei/8c2c2b98554b32f7dd5fc9bc44a6b1da42f117c19ee97f5a55c9b74d4d33c7cd";
```

Comment:
Don't commit the rpc address, fetch them from environment variables

Reply:
Roger that!